### ISSUE TITLE
Can't find moq.autocomplete extension in the Visual Studio Marketplace

### ISSUE THREAD

**Issue by cezarypiatek** [Litee/moq.autocomplete#1]
Hi,

I would like to use this extension but can't find it in the  Visual Studio Marketplace. The version distributed as a nuget package doesn't work - I've installed `moq.autocomplete` from nuget but I can't get any suggestions. Would you mind to publish `moq.autocomplete` as a vsix via the official channel with enabled support for Visual Studio 2019?

**Comment by Litee** [comment]
Hey!
I tried to onboard to the Marketplace some time ago, but the site was giving me an internal error for some reason. Let me try again.

Also, I can confirm that autocomplete from NuGet package is not working anymore for some reason.   I am not using Moq at the moment so don't want to spend time investigating this - VSIX should be enough.

**Comment by Litee** [comment]
New version of the extension was uploaded to the Marketplace

**Comment by cezarypiatek** [comment]
@Litee thanks for uploading the new version to the marketplaces. Unfortunately, I have a bad news -  I've tested it and it doesn't work. I'm not able to get any suggestions from `Moq.Autocomplet`.

![image](https://user-images.githubusercontent.com/7759991/62010700-62959580-b16e-11e9-9693-037ab4bdf22d.png)

I tried to debug it but after building and running this extension I'm getting an error about loading MEF extension.

**Comment by Litee** [comment]
Hmm, I checked them myself and they worked. Could be a dependency on some SDK installed into the system. Do you see what exactly fails to load?

**Comment by Litee** [comment]
![image](https://user-images.githubusercontent.com/400234/62070879-47d92480-b233-11e9-973f-e32e521f7474.png)
![image](https://user-images.githubusercontent.com/400234/62070909-56bfd700-b233-11e9-8022-3cb13afc9cce.png)

**Comment by cezarypiatek** [comment]
I've spent a few hours trying to debug this extension. I used dnspy and attached debugger to the Visual Studio. It looks like the extension is load correctly, `ProvideCompletionsAsync` are invoked but in the code below:

```cs
SyntaxToken tokenAtCursor = Helpers.GetCurrentArgumentListSyntaxToken(syntaxRoot, context.Position);
```

`tokenAtCursor` are always None. What version of the VS did you use to test the extension?

**Comment by Litee** [comment]
![image](https://user-images.githubusercontent.com/400234/62082480-c0001400-b24c-11e9-9aa5-822ddb19c584.png)

**Comment by cezarypiatek** [comment]
Ok, I've observed another thing: when I trigger completion with `ctr+j` the suggestions are working but when I trigger the completion with `ctr+space` there is no suggestion from Moq.Autocomplete. It seems that for  `ctr+space` visual studio is passing the wrong value for `context.Position`

**Comment by Litee** [comment]
Is your VS version the same?

**Comment by cezarypiatek** [comment]
![image](https://user-images.githubusercontent.com/7759991/62222736-23f11c80-b3b4-11e9-932e-8d6d64973d89.png)
